noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable/test.nr

```diff
@@ -13,336 +16,445 @@ global new_delay: u64 = 20;
 
 global storage_slot: Field = 47;
 
-global TEST_INITIAL_DELAY: u64 = 32;
+global TEST_INITIAL_DELAY: u64 = 60 * 60 * 24;
 
 unconstrained fn setup() -> TestEnvironment {
-    TestEnvironment::new()
+    TestEnvironment::_new()
 }
 
 unconstrained fn in_public(
```

Comment:
I don't think these functions are useful anymore, reading `env.public_context(|context| let state_var = in_public(context)...` feels redundant

Reply:
`SharedMutable` in particular is a bit annoying because the type definition also includes the delay, I kept them for the purpose of not having to explicitly type out the type (heh) in each test. But agreed, could maybe reduce the boilerplate a little bit. I like the tests being explicit though.